packages/providers/fal/text-to-video/fal_text_to_video/cli.py

```
@@ -119,164 +133,59 @@ def cmd_model_info(args):
 
         print(f"\nFeatures:")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)